dataschema/Schema.proto

```diff
@@ -203,6 +203,9 @@ message ColumnClickhouseAnnotation {
   // Override the name of the column when generating SQL columns
   // (in clickhouse or via SQL alchemy).
   optional string original_name = 5;
+  // Override the default type for a nested field with this ClickHouse type
```

Comment:
Want to have this as a string, or a bool like nested_use_tuple ?

Reply:
I thought string would be a good choice in case CH provides more nested data types in the future besides just Nested and Tuple.

Do you have a preference?

Reply:
Ok  - in this case please add a check in annotations.py that the nested_type_name is in a well reseticted set - for now just {'Tuple'}

Reply:
Done, added the check and test.